src/ui/FieldSuggestions/FieldSuggestions.ts

```diff
@@ -134,13 +164,15 @@ export class FieldSuggestions extends Component {
     this.bind.onRootElement(OmniboxEvents.populateOmnibox, (args: IPopulateOmniboxEventArgs) => this.handlePopulateOmnibox(args));
   }
 
-  /**
-   * Select a currently displayed suggestion. This means that at least one suggestion must have been returned at least once.
-   * The suggestion parameter can either be a number (0 based index of the suggestion to select) or a string that match the suggestion.
-   * @param suggestion
-   */
   public selectSuggestion(suggestion: number);
   public selectSuggestion(suggestion: string);
+
```

Comment:
I unexposed the `selectSuggestion(suggestion: number)` method and exposed the `selectSuggestion(suggestion: any)` override to cover both number and string cases in typing. Is that OK?